src/workload.py

```diff
@@ -116,3 +117,32 @@ def generate_password(self) -> str:
             String of 32 randomized letter+digit characters
         """
         return "".join([secrets.choice(string.ascii_letters + string.digits) for _ in range(32)])
+
+    @override
+    def get_version(self) -> str:
+        bash_netcat = (
+            f"echo '4lw' | (exec 3<>/dev/tcp/localhost/{CLIENT_PORT}; cat >&3; cat <&3; exec 3<&-)"
+        )
+        ruok = [bash_netcat.replace("4lw", "ruok")]
+        stat = [bash_netcat.replace("4lw", "stat")]
```

Comment:
**suggestion** Uhm, maybe I would put the `ruok` elsewhere (if we have a health check) rather than here. We can then put the timeout (+ retry logic maybe) there, and leave this method rather slim. 

I usually use "srvr" to print the server information, among which the version, but up to you.

Reply:
841363f02de204874d08b5580723765f74d1357d now uses `.healthy`!